### ISSUE TITLE
items url automation

### ISSUE THREAD

**Issue by samcome** [mirtilb/NFT-Opensea-Bulk-Pricing#1]
Hello, I plan to test your script but I was asking how we can retrieve the urls of items contained in a collection. It is impossible to retrieve the urls of 5000 items without automation ;) If you have a solution?
Thank you!

**Comment by mirtilb** [comment]
Sure! 
After creating your collection, click on the 1st item and you'll get an url like this one:
https://opensea.io/assets/0x43fb98305641c8aa1514ff15e00e007df02f711e/36

Here, you can see:
- url: https://opensea.io/assets/
- collection id: 0x43fb98305641c8aa1514ff15e00e007df02f711e
- 36: collectible id

So
You just need to create a XLSX with as many lines as you need where 
- 1st column: https://opensea.io/assets/0x43fb98305641c8aa1514ff15e00e007df02f711e/
- 2nd column: id from 1 to 10000
then you export as a csv, open it and remove all commas. Done all you urls are ready ;-)

**Comment by samcome** [comment]
Yes I understood the operation well, the question was how to retrieve item urls automatically without retrieving them by hand one by one.

https://opensea.io/assets/0x495f947276749ce646f68ac8c248420045cb7b5e/19725514370854135845209296410799779181823946741638112438966970721276386082817

ID is: 19725514370854135845209296410799779181823946741638112438966970721276386082817

**Comment by mirtilb** [comment]
ha ok, understood: you didn't create your custom smart contract but used the OPENSEA solution to mint NFTs into their "OpenSea Shared Storefront (OPENSTORE)": https://etherscan.io/address/0x495f947276749ce646f68ac8c248420045cb7b5e

the issue here is that your collection is mixed with everyone else, so id are attributed randomly so no real solution for that sorry :( 

It would be better for your collection management and collectible values to use solution like CARGO to create your own smart contract with your 10k items inside (estimated budget on ETH blockchain: ~$700... depends of gas price).

**Comment by samcome** [comment]
I understand, i'll watch Cargo. Thank you! :)
